contracts/contracts/core/ProviderRegistry.sol

```diff
@@ -212,6 +212,17 @@ contract ProviderRegistry is
         emit Withdraw(msg.sender, providerStake);
     }
 
+    /**
+     * @dev Allows the bidder to withdraw the slashed amount.
+     */
+    function bidderWithdraw() external nonReentrant whenNotPaused() {
```

Comment:
This is specifically for withdrawing slashed amount, correct? If so the naming should reflect that. `withdrawSlashedAmount` etc. 

Also why is this function needed, ie. which bug does it fix again?

Reply:
Got it, sounds good. Just the naming of the function is relevant from my original comment then